web/src/lib/utils/analytics.ts

```diff
@@ -123,7 +123,7 @@ type StatusSelected = "selected_not_started" | "selected_in_progress" | "selecte
 
 type Phase =
   | "progress_to_needs_to_register_phase"
```

Comment:
[pebble] We should remove the analytics for `"progress_to_needs_to_register_phase"` since we are removing the `NEEDS TO REGISTER` operating phase.